docker-compose.yml

```diff
@@ -120,6 +120,7 @@ services:
       - HONEYCOMB_SERVICE=od2
       - HONEYCOMB_WRITEKEY=buzzzzzzzzzzzzzzzz
       - IIIF_SERVER_BASE_URL=http://localhost:8080/iiif
+      - HOME=/data
```

Comment:
I don't think this is necessary - it should already be fine in the image, and it shouldn't need to be changed when running the container